data-raw/dm_ped.R

```diff
@@ -0,0 +1,74 @@
+# Dataset: dm_ped
+# Description: Create DM test SDTM dataset for pediatric studies
+
```

Comment:
Please add code to load required packages (dplyr etc)

Reply:
added dplyr and purrr - this should be sufficient - please let me know otherwise